### PR TITLE
fix: Datepicker renders input description if inputProps.onRenderDescription is set

### LINKED ISSUE #27021: [Bug]: (DatePicker) props.textField.onRenderDescription is not invoked
### Library

React / v8 (@fluentui/react)

### System Info

```shell
npmPackages:
    @fluentui/react: 8.106.1
    @fluentui/react-hooks: 8.6.18
  System:
    OS: Windows 10 10.0.19045
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11850H @ 2.50GHz
    Memory: 11.60 GB / 31.73 GB
  Browsers:
    Chrome: 110.0.5481.177
    Edge: Spartan (44.19041.1266.0), Chromium (110.0.1587.57)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codepen.io/bestadd/pen/VwGpmGv

## Actual Behavior
DatePicker's props.textField.onRenderDescription is never called

## Expected Behavior
If specified props.textField.onRenderDescription is invoked and the returend value is rendered as the the DatePicker's Description.

## Notes
This seams to have been introduced by commit e5fb795ee7d65b344d26c227cfa83b0da05c0512

The internal renderTextfieldDescription function needs to see if a value has been provided for props.textField.onRenderDescription and invoke it rather then the defaultRender.

I am hoping someone here is willing to make this supper small change, I'm currently working with my legal team to gain authorization to make contributions here, but currently I'm unable to. 

### Logs

```shell
There is no relevant log output
```

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by smhigley**
## Description
Datepicker respected `inputProps.description`, but not `inputProps.onRenderDescription` on its own. Now both props individually work to render the description

## Related Issue(s)

- Fixes #27021

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659'>3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-75983ba2-b5ea-4e4e-9578-6fc0293ce309.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-75983ba2-b5ea-4e4e-9578-6fc0293ce309.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27784&commitId=50caed327aa751c7a75c841e52aaf5d2ab75d8c2&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/BaseButton-mount_base.html">818</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/BaseButton-mount.html">823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Breadcrumb-mount_base.html">2200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Breadcrumb-mount.html">2202</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Checkbox-mount_base.html">2139</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Checkbox-mount.html">2146</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/CheckboxBase-mount_base.html">1981</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/CheckboxBase-mount.html">1890</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ChoiceGroup-mount_base.html">3789</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ChoiceGroup-mount.html">3712</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ComboBox-mount_base.html">881</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ComboBox-mount.html">881</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/CommandBar-mount_base.html">8047</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/CommandBar-mount.html">8050</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ContextualMenu-mount_base.html">17122</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ContextualMenu-mount.html">19600</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DefaultButton-mount_base.html">991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DefaultButton-mount.html">1005</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DetailsRow-mount_base.html">2910</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DetailsRow-mount.html">2927</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DetailsRowFast-mount_base.html">2940</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DetailsRowFast-mount.html">2888</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DetailsRowNoStyles-mount_base.html">2677</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DetailsRowNoStyles-mount.html">2668</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Dialog-mount_base.html">3456</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Dialog-mount.html">3446</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DocumentCardTitle-mount_base.html">312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/DocumentCardTitle-mount.html">326</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Dropdown-mount_base.html">2587</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Dropdown-mount.html">2581</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/FocusTrapZone-mount_base.html">1505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/FocusTrapZone-mount.html">1535</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/FocusZone-mount_base.html">1501</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/FocusZone-mount.html">1483</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedList-mount_base.html">50681</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedList-mount.html">58543</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedList-virtual-rerender_base.html">24449</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedList-virtual-rerender.html">24380</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">75073</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedList-virtual-rerender-with-unmount.html">74768</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedListV2-mount_base.html">312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedListV2-mount.html">316</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedListV2-virtual-rerender_base.html">297</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedListV2-virtual-rerender.html">297</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">321</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/IconButton-mount_base.html">1436</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/IconButton-mount.html">1420</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Label-mount_base.html">456</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Label-mount.html">469</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Layer-mount_base.html">3637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Layer-mount.html">3585</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Link-mount_base.html">533</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Link-mount.html">523</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/MenuButton-mount_base.html">1248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/MenuButton-mount.html">1217</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/MessageBar-mount_base.html">27739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/MessageBar-mount.html">27431</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Nav-mount_base.html">2543</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Nav-mount.html">2528</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/OverflowSet-mount_base.html">1059</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/OverflowSet-mount.html">1025</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Panel-mount_base.html">2265</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Panel-mount.html">2251</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Persona-mount_base.html">969</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Persona-mount.html">966</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Pivot-mount_base.html">1180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Pivot-mount.html">1178</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/PrimaryButton-mount_base.html">1113</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/PrimaryButton-mount.html">1098</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Rating-mount_base.html">5765</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Rating-mount.html">5850</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SearchBox-mount_base.html">1178</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SearchBox-mount.html">1132</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Shimmer-mount_base.html">2343</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Shimmer-mount.html">2373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Slider-mount_base.html">1732</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Slider-mount.html">1706</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SpinButton-mount_base.html">3730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SpinButton-mount.html">3769</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Spinner-mount_base.html">523</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Spinner-mount.html">522</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SplitButton-mount_base.html">2457</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SplitButton-mount.html">2426</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Stack-mount_base.html">533</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Stack-mount.html">526</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/StackWithIntrinsicChildren-mount_base.html">1134</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/StackWithIntrinsicChildren-mount.html">1128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/StackWithTextChildren-mount_base.html">3246</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/StackWithTextChildren-mount.html">3256</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SwatchColorPicker-mount_base.html">8187</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/SwatchColorPicker-mount.html">8152</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/TagPicker-mount_base.html">1995</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/TagPicker-mount.html">2010</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Text-mount_base.html">505</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Text-mount.html">496</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/TextField-mount_base.html">1198</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/TextField-mount.html">1199</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ThemeProvider-mount_base.html">1145</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ThemeProvider-mount.html">1137</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ThemeProvider-virtual-rerender_base.html">776</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ThemeProvider-virtual-rerender.html">774</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1725</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1739</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Toggle-mount_base.html">806</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/Toggle-mount.html">814</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/buttonNative-mount_base.html">289</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27784/perf-test/buttonNative-mount.html">280</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by sopranopillow — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-react-75983ba2-b5ea-4e4e-9578-6fc0293ce309.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-75983ba2-b5ea-4e4e-9578-6fc0293ce309.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=27784&commitId=7fb9b8eda445557579b42cf5547e9a96c1550781&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Pull request merged**